libs/labelbox/tests/data/annotation_import/conftest.py

```diff
@@ -696,6 +768,42 @@ def _create_prompt_response_project(client: Client, rand_gen, media_type, normal
     
     return prompt_response_project, ontology
 
+def _create_offline_mmc_project(client: Client, rand_gen, data_row_json, normalized_ontology) -> Tuple[Project, Ontology, Dataset]:
+    """ Shared function to configure project for integration tests """
```

Comment:
Can you update this doc string to something more accurate or remove it altogether?

Reply:
oh yes, of course